oauth2_http/java/com/google/auth/oauth2/ExternalAccountAuthorizedUserCredentials.java

```diff
@@ -329,6 +330,7 @@ static ExternalAccountAuthorizedUserCredentials fromJson(
         .setRefreshToken(refreshToken)
         .setHttpTransportFactory(transportFactory)
         .setQuotaProjectId(quotaProjectId)
+        .setUniverseDomain(universeDomain)
```

Comment:
nit: defaulting logic? Or is that handled in a parent class already

Reply:
Yes, defaults to GDU in GoogleCredentials.